verification/infrastructure/repositories/individual_repository.py

```diff
@@ -0,0 +1,47 @@
+from common.logger import get_logger
+from verification.domain.factory.verification_factory import VerificationFactory
+from verification.infrastructure.models import IndividualVerificationModel
+from verification.infrastructure.repositories.base_repository import BaseRepository
+
+logger = get_logger(__name__)
+
+
+class IndividualRepository(BaseRepository):
+
+    def add_verification(self, verification):
+        verification_db = IndividualVerificationModel(
+            verification_id=verification.verification_id, username=verification.username,
+            comments=verification.comment_dict_list(),
```

Comment:
We should have comments as an attribute in the verification model.
comments=verification.comments

Reply:
while storing the comments in domain entity we will use domain object list of `Comment`. and when we use in application layer or the repo layer we will make it `dict`.